techlottus/portalverse
# Remove paragraphs from Footer

## Issue
Remove paragraphs from Footer

## Solution
- [X] Remove paragraphs from Footer 4b42a04
## Testing
1. Go to FooterPortalVerse.tsx
2. Go to the lines 102 and 103
3. See the comments
4. Enjoy 🌮
